mlrun/feature_store/api.py

```diff
@@ -322,6 +323,17 @@ def _rename_source_dataframe_columns(df):
     return df
 
 
+def validate_graph_steps(featureset: FeatureSet):
+    if DropFeatures.__name__ in featureset.graph.steps:
+        entities_keys = [entity.name for entity in featureset.spec.entities]
```

Comment:
does this function supposed to be a user facing function, if not lets change it to be a private function.

Reply:
nice idea! tnx